backend/judging-algorithm/data/schemas/judges-schema.ts

```diff
@@ -3,7 +3,11 @@ import mongoose from "mongoose";
 export const judgesSchema = new mongoose.Schema(
   {
     name: { type: String, required: true },
+    inPerson: { type: Boolean, required: true}
+  },
+  { 
+    collection: "Judges",
+    versionKey: false
```

Comment:
why's versionKey required?

Reply:
without versionKey set to false, if we create objects for our database it adds an additional versionKey field in the data

Reply:
Ohh ok, thanks!!